pkgs/development/tools/wrangler/default.nix

```diff
@@ -13,16 +13,21 @@ rustPlatform.buildRustPackage rec {
 
   cargoSha256 = "0w845virvw7mvibc76ar2hbffhfzj2v8v1xkrsssrgzyaryb48jk";
 
-  nativeBuildInputs = [ perl ] ++ lib.optionals stdenv.isLinux [ pkg-config ];
+  nativeBuildInputs = [ pkg-config ];
 
-  buildInputs = lib.optionals stdenv.isLinux [ openssl ]
+  buildInputs = [ openssl ]
     ++ lib.optionals stdenv.isDarwin [
       curl
       darwin.apple_sdk.frameworks.Security
       darwin.apple_sdk.frameworks.CoreServices
       darwin.apple_sdk.frameworks.CoreFoundation
     ];
 
+  cargoBuildFlags = [
```

Comment:
Might be easier to use `OPENSSL_NO_VENDOR` for this?

Reply:
Ah, yes, great idea.